x/vm/types/preinstall.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+import "github.com/ethereum/go-ethereum/common"
+
+type Preinstall struct {
+	Address common.Address
```

Comment:
Could we add names here to keep track of what these are?

Reply:
Yeah, I think that is a good idea. I will be doing that!

Reply:
Done!